ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -548,3 +522,36 @@ func getClusterSize(context *cli.Context) (string, error) {
 
 	return size, nil
 }
+
+// If param1 exists, param2 is not allowed.
+func validateMutuallyExclusiveParams(cfnParams *cloudformation.CfnStackParams, param1, param2 string) bool {
```

Comment:
the names of these functions don't indicate which state they're asserting (they seem to return 'true' if an error state is detected) - what do you think about renaming this to something like "assertMutuallyExclusiveParamsPresent" ? Or do you think this work beyond the scope of this PR?

Reply:
This was just a move, so was otherwise going to leave this untouched